lib/opentracing.rb

```diff
@@ -0,0 +1,24 @@
+require "forwardable"
+require "opentracing/version"
+require "opentracing/span_context"
+require "opentracing/span"
+require "opentracing/tracer"
+
+module OpenTracing
+  # Text format for #inject and #extract
+  FORMAT_TEXT_MAP = 1
+
+  # Binary format for #inject and #extract
+  FORMAT_BINARY = 2
+
+  # Ruby Specific format to handle how Rack changes environment variables.
+  # See Readme.md for more info.
+  FORMAT_RACK = 3
+
+  class << self
```

Comment:
I'm not sure if the ability to do `OpenTracing.start_span` is consistent with other languages. @bensigelman would know better.